csug/syntax.stex

```diff
@@ -574,6 +574,9 @@ With two arguments, \var{id} and \var{key}, \var{lookup} returns the
 value of  \var{id}'s \var{key} property, or \scheme{#f} if \var{id}
 has no \var{key} property.
 
+However, in the case of properties, there is a second, usually easier
+way, to obtain property values, namely by calling the
+\scheme{property-value} procedure described below.
```

Comment:
For compile-time properties, the \scheme{property-value} procedure can be used instead of the lookup procedure in macro transformers.

Reply:
The best wording probably depends on what the "recommended" way to retrieve property values should be. With `property-value` being available, I see no reason to use the older way for new code.